services/user.py

```diff
@@ -0,0 +1,52 @@
+from django.contrib.auth import get_user_model
+
+
+User = get_user_model()
+
+
+def create_user(
+    username: str,
+    password: str,
+    email: str = None,
+    first_name: str = None,
+    last_name: str = None,
+) -> User:
+    user = User.objects.create_user(username=username)
```

Comment:
This line of code does not meet the requirement 'Choose correct and suitable variable names' from the checklist. The variable `user` is suitable in this context, but remember to always use descriptive and meaningful variable names.